tests/testthat/test-families.R

```diff
@@ -35,6 +35,6 @@ test_that("all combinations run without errors", {
     sfit <- do.call(sgdnet, pars)
     gfit <- do.call(glmnet, pars)
 
-    compare_predictions(sfit, gfit, x, tol = 1e-3)
+    compare_predictions(sfit, gfit, x, tol = 1e-2)
```

Comment:
Is this necessary? If only needed for the Poisson, maybe special case it? I don't like loosening tolerances unless we really have to.